src/index.ts

```diff
@@ -101,6 +101,22 @@ export class Amplitude {
     return AmplitudeReactNative.getDeviceId(this.instanceName);
   }
 
+  /**
+   * Use the Advertising ID on Android if available from Google Play Services.
+   * Must be called before init.
+   */
+  advertisingIdForDeviceId(): Promise<boolean> {
```

Comment:
I think we can expose the useAdvertisingIdForDeviceId to the customer but use  `advertisingIdForDeviceId` internally?  or replace both to setAdvertisingIdAsDeviceId? What do you think?

Reply:
I think we should use the most consistent name to prevent confusion and also strange errors like the one we just talked about when we create a method `use...` I'll change it to `setAdvertisingIdAsDeviceId` and `setAppSetIdAsDeviceId` wherever possible